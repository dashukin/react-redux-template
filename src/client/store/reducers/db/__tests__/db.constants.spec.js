import dbConstants from '../db.constants';

describe('db.constants', () => {
  it('should match snapshot', () => {
    expect(dbConstants).toMatchSnapshot();
  });
});
