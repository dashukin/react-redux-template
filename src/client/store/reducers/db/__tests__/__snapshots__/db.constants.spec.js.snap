// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`db.constants should match snapshot 1`] = `
Object {
  "DB_ADD": "DB_ADD",
}
`;
